File: app/src/main/java/com/nelyan/ui/ItemsAdapter.java
package com.nelyan.ui;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.nelyan.R;

public class ItemsAdapter extends RecyclerView.Adapter<ItemsAdapter.MyViewHolder> {
    Context context;

    private LayoutInflater inflater;

    public ItemsAdapter(Context applicationContext) {
        this.context = applicationContext;

    }

    @Override
    public MyViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {

        inflater = LayoutInflater.from(viewGroup.getContext());
        View view = inflater.inflate(R.layout.res, viewGroup, false);

        return new MyViewHolder(view);
    }

    @Override
    public void onBindViewHolder(@NonNull final MyViewHolder myViewHolder, final int i) {

    }

    // Return the type of your itemsData (invoked by the layout manager)

    @Override
    public int getItemCount() {

        return 4;
    }

    public class MyViewHolder extends RecyclerView.ViewHolder {


        public MyViewHolder(@NonNull View itemView) {
            super(itemView);

        }
    }
}

File: app/src/main/java/com/nelyanlive/ui/ItemsAdapter.kt
package com.nelyanlive.ui

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.recyclerview.widget.RecyclerView
import com.nelyanlive.R

class ItemsAdapter(var context: Context) : RecyclerView.Adapter<ItemsAdapter.MyViewHolder>() {
    private var inflater: LayoutInflater? = null
    override fun onCreateViewHolder(viewGroup: ViewGroup, i: Int): MyViewHolder {
        return MyViewHolder( LayoutInflater.from(viewGroup.context).inflate(R.layout.res, viewGroup, false))
    }

    override fun onBindViewHolder(myViewHolder: MyViewHolder, i: Int) {}

    // Return the type of your itemsData (invoked by the layout manager)
    override fun getItemCount(): Int {
        return 5
    }

    inner class MyViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView)

}